Day 2/Day 2 Solution.ipynb

```diff
@@ -0,0 +1,179 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "id": "c61a6603",
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "AttributeError",
+     "evalue": "'str' object has no attribute 'type'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[1;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
+      "\u001b[1;32m<ipython-input-30-3aedf3ee4047>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m     10\u001b[0m \u001b[0mnewsteps\u001b[0m \u001b[1;33m=\u001b[0m \u001b[1;33m[\u001b[0m\u001b[0mi\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;36m0\u001b[0m\u001b[1;33m]\u001b[0m \u001b[1;32mfor\u001b[0m \u001b[0mi\u001b[0m \u001b[1;32min\u001b[0m \u001b[0msteps\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     11\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 12\u001b[1;33m \u001b[0mnewsteps\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;36m0\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mtype\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     13\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
+      "\u001b[1;31mAttributeError\u001b[0m: 'str' object has no attribute 'type'"
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "import csv\n",
+    "import csv\n",
+    "\n",
+    "with open('input2.csv', newline='') as f:\n",
+    "    reader = csv.reader(f)\n",
+    "    steps = list(reader)\n",
+    "\n",
+    "    \n",
+    "newsteps = [i[0] for i in steps]\n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 100,
+   "id": "c33dd74d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "horizontal = 0\n",
+    "depth = 0\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 93,
+   "id": "00fcbb3a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "forward\n",
+      "2\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(newsteps[0].split()[0])\n",
+    "print(newsteps[0].split()[1])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 101,
+   "id": "d8ffeb04",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for i in newsteps:\n",
+    "    if i.split()[0] == 'forward':\n",
+    "        horizontal += int(i.split()[1])\n",
+    "    elif i.split()[0] == 'down':\n",
+    "        depth += int(i.split()[1])\n",
+    "    elif i.split()[0] == 'up':\n",
+    "        depth -= int(i.split()[1])\n",
+    "    else:\n",
+    "        print('nerd')"
```

Comment:
This is quite inappropriate, please change to `error`.

Reply:
changes made :)